projects/fsastorefrontstyles/scss/_fs-theme.scss

```diff
@@ -18,10 +18,11 @@ $fsSelectors: cx-fs-policy-details, cx-fs-claim-details, cx-fs-premium-calendar,
   cx-fs-update-profile-form, cx-fs-product-assignments,
   cx-fs-potential-product-assignments, cx-fs-login-form, cx-update-email,
   cx-update-password, cx-fs-active-products, cx-fs-change-coverage,
-  cx-form-component, cx-fs-select-identification, cx-fs-add-options,
-  cx-fs-cart-coupon, cx-fs-payment-method, cx-fs-order-details-totals,
-  cx-fs-order-overview, cx-fs-order-details-items, cx-fs-order-history,
-  cx-fs-message-notification, cx-fs-documents-table !default;
+  cx-form-component, cx-dynamic-form, cx-fs-select-identification,
```

Comment:
Why do we have now "cx-form-component" and "cx-dynamic-form" when we can have all dynamicforms styling under "cx-form-component"?

Reply:
My reasoning was that as far as I could see `cx-form-component` is just a wrapper that we are using for styling box-shadow and form title. I thought it would be more flexible to have that separate and as optional, so end-user would not have to override it with their styling, just turn it on or off. This way they could keep the other styling for forms' inner content. Seems a bit more flexible?

Reply:
I would suggest leaving it as it was before, since "cx-form-component" is the main wrapper for whole dynamic forms. 
That way we keep it simple for end user to toggle our styling, by skipping styles with one selector.